src/main/java/uk/gov/hmcts/reform/draftstore/domain/UpdateDraft.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.draftstore.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import uk.gov.hmcts.reform.draftstore.service.validation.JsonObject;
+
+import javax.validation.constraints.NotNull;
+
+public class UpdateDraft {
```

Comment:
Besides name, there are no differences to `CreateDraft` class. What is the responsibility? Maybe this data structure can be named and re-used?

Reply:
I actually like this separation. While the structure is the same, UpdateDraft and CreateDraft serve different requests and can be considered unrelated.